packages/mds-ingest-service/tests/index.spec.ts

```diff
@@ -550,6 +550,17 @@ describe('Ingest Service Tests', () => {
         type: 'ConflictError'
       })
     })
+
+    it('gets events with 2 annotations', async () => {
+      await IngestRepository.createDevices([TEST_TNC_A, TEST_TNC_B])
+      await IngestRepository.createEvents([TEST_EVENT_A1, TEST_EVENT_B1])
+      await IngestRepository.createEvents([TEST_EVENT_A2, TEST_EVENT_B2])
+      const { events } = await IngestServiceClient.getEventsUsingOptions({
+        time_range: { start: testTimestamp, end: testTimestamp + 2000 },
+        grouping_type: 'all_events'
+      })
+      expect(events.filter(e => e.annotation).length).toEqual(2)
```

Comment:
I'm actually kind of surprised that annotations _are_ coming back here (createEvents doesn't generate annotations directly IIRC), are they `null`? Would be nice to have a test where there's content for the annotation.

Reply:
They are non-null and have valid content.
<img width="504" alt="Screen Shot 2021-07-27 at 7 22 29 AM" src="https://user-images.githubusercontent.com/1615977/127171046-2d78f2b9-93bb-4a4c-b99f-6a6d5a201fa9.png">

Reply:
Ah I see, those annotations were already written in during a previous test `'writes two event annotations'`. Side effects woooo

Reply:
Good catch, fixed